hack/go-genmock.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# Example:  ./hack/go-genmock.sh
+
+if [ "$IS_CONTAINER" != "" ]; then
+  go generate ./pkg/... "${@}"
```

Comment:
Should ./pkg/... be taken as argument? Like in ./hack/go-lint.sh !

Reply:
It could be. But I would argue that it is better to have the knowledge for where the files are that need to have generation run against in the script rather than relying on the individual calling the script to add it.

Reply:
I was only thinking the case where someone may need to override it and there isn't a way.

Reply:
If a user has a short-term reason to want to override something, the script can be edited locally.

Reply:
fair enough